RaidBot/Entities/Raid.cs

```diff
@@ -54,13 +64,31 @@ public int UserCount
             raidWithUsers.Append("No users are attending this raid");
          }
          else {
-            raidWithUsers.Append($"Leader: {Users.First().Username} {Users.First().GetGuests} | ");
+            raidWithUsers.Append($"Leader: {Users.First().Username} {Users.First().GetGuests}");
+
+            List<User> inPersonUsers = new List<User>();
+            List<User> remoteUsers = new List<User>();
 
-            foreach (var user in Users.Skip(1)) {
-               raidWithUsers.Append($"{user.Username} {user.GetGuests} | ");
+            foreach (var user in Users) {

```

Comment:
There is grouping you can do in LINQ. Not necessary, but just wanted to point that out!

Reply:
👍 Thanks!